views/components/collapsibleBlock.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * All About Olaf
+ * Individual row for inside a collapsible view
+ */
+
+import React from 'react'
+import {
+  View,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native'
+
+import Collapsible from 'react-native-collapsible'
+
+
+export default class CollapsibleBlock extends React.Component {
+
+  static propTypes = {
+    backgroundColor: React.PropTypes.string.isRequired,
+    children: React.PropTypes.array.isRequired,
+  }
+
+  state = {
+    isCollapsed: true,
+  }
+
+  styles = StyleSheet.create({
+    container: {
+      marginBottom: 0,
+    },
+    collapsedContent: {
+      alignSelf: 'center',
+
+    },
+    collapsedContainer: {
+    },
+  })
+
+  render() {
+    return (
+        <TouchableOpacity
+          onPress={() => this.setState({isCollapsed: !this.state.isCollapsed})}
+        >
+          {this.props.children[0]}
```

Comment:
There should be exactly 2 children in the array. One is the TouchableOpacity "header" and the other is the dropdown contents